regression/scripts/Vehicles/Boeing_737.py

```diff
@@ -83,391 +73,399 @@ def base_setup():
     #   Flaps
     # ------------------------------------------------------------------
     wing.flaps.chord      =  0.30   
     wing.flaps.span_start =  0.10   # ->     wing.flaps.area = 97.1112
     wing.flaps.span_end   =  0.75
     wing.flaps.type       = 'double_sloted'  # -> wing.flaps.number_slots = 2
-
-    # wing sizing
-    SUAVE.Methods.Geometry.Two_Dimensional.Planform.wing_planform(wing)
-
+    
+    
     # add to vehicle
     vehicle.append_component(wing)
 
 
     # ------------------------------------------------------------------        
     #  Horizontal Stabilizer
     # ------------------------------------------------------------------        
-
+    
     wing = SUAVE.Components.Wings.Wing()
     wing.tag = 'horizontal_stabilizer'
-
-    wing.aspect_ratio            = 6.16
+    
+    wing.aspect_ratio            = 6.16      #
     wing.sweeps.quarter_chord    = 30 * Units.deg
     wing.thickness_to_chord      = 0.08
     wing.taper                   = 0.4
     wing.span_efficiency         = 0.9
-
-    wing.spans.projected         = 14.146
+    
+    wing.spans.projected         = 14.146      #
 
     wing.chords.root             = 3.28
     wing.chords.tip              = 1.31    
     wing.chords.mean_aerodynamic = 8.0
 
-    wing.areas.reference         = 32.488
-
+    wing.areas.reference         = 32.488    #
+    wing.areas.exposed           = 199.7792                  # Exposed area of the horizontal tail
+    wing.areas.wetted            = 249.724                   # Wetted area of the horizontal tail
     wing.twists.root             = 3.0 * Units.degrees
     wing.twists.tip              = 3.0 * Units.degrees  
-
+    
     wing.origin                  = [50,0,0]
     wing.aerodynamic_center      = [2,0,0]
-
+    
     wing.vertical                = False 
     wing.symmetric               = True
-
+    
     wing.dynamic_pressure_ratio  = 0.9  
-
+    
     # add to vehicle
     vehicle.append_component(wing)
-
-
+    
+    
     # ------------------------------------------------------------------
     #   Vertical Stabilizer
     # ------------------------------------------------------------------
-
+    
     wing = SUAVE.Components.Wings.Wing()
     wing.tag = 'vertical_stabilizer'    
-
-    wing.aspect_ratio            = 1.91
+    
+    wing.aspect_ratio            = 1.91      #
     wing.sweeps.quarter_chord    = 25 * Units.deg
     wing.thickness_to_chord      = 0.08
     wing.taper                   = 0.25
     wing.span_efficiency         = 0.9
-
-    wing.spans.projected         = 7.877
+    
+    wing.spans.projected         = 7.877      #    
 
     wing.chords.root             = 6.60
     wing.chords.tip              = 1.65
     wing.chords.mean_aerodynamic = 8.0
-
-    wing.areas.reference         = 32.488
-
+    
+    wing.areas.reference         = 32.488    #
+    
     wing.twists.root             = 0.0 * Units.degrees
     wing.twists.tip              = 0.0 * Units.degrees  
-
+    
```

Comment:
Some of these values are old. Check to see what's in the tutorial B737. That is more up to date. Since we need to build off these in the future, it's probably best to use the most correct info.